test-loop-tests/src/utils/peer_manager_actor.rs

```diff
@@ -52,6 +56,15 @@ pub struct ViewClientSenderForTestLoopNetwork {
     pub block_request: AsyncSender<BlockRequest, ActixResult<BlockRequest>>,
 }
 
+/// This message is used to allow TestLoopPeerManagerActor to construct NetworkInfo for each
+/// client.
+#[derive(actix::Message, Debug, Clone, PartialEq, Eq)]
```

Comment:
Could we add an additional comment on who sends this info to whom and at what interval?